dev/TabView/TabView.idl

```diff
@@ -7,6 +7,16 @@ enum TabViewWidthMode
 {
     Equal = 0,
     SizeToContent = 1,
+    Compact = 2,
+};
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+enum TabViewCloseButtonOverlayMode
+{
+    Auto = 0,
+    OnHover = 1,
+    Persistent = 2,
 };
```

Comment:
The community toolkit had a bool instead of an enum here. @stmoy, can you please comment on the desired API?

Reply:
I'll include this in the API spec. I think enum makes more sense here, particularly because it supports the Auto value (per my comment here: https://github.com/microsoft/microsoft-ui-xaml/issues/2007#issuecomment-590506160